src/dlgProfilePreferences.cpp

```diff
@@ -183,6 +183,7 @@ dlgProfilePreferences::dlgProfilePreferences(QWidget* pF, Host* pH) : QDialog(pF
     for (int i = 1; i < 40; i++) {
         sizeList << QString::number(i);
     }
+    // this seems to attempt adding an empty index at 0, but doesn't seem to work as expected.
```

Comment:
Are you sure? The documentation is:
> `void QComboBox::insertItems(int index, const QStringList &list)`
Inserts the strings from the list into the combobox as separate items, starting at the index specified.
If the index is equal to or higher than the total number of items, the new items are appended to the list of existing items. If the index is zero or negative, the new items are prepended to the list of existing items.
See also `insertItem()`.

So the following code line should fill out (by appending) the _previously_ __empty__ `QComboBox` with the numbers from 1 to 39 inclusive in that order...